src/Akka.Persistence.Azure/Journal/AzureTableStorageJournalSettings.cs

```diff
@@ -75,10 +79,14 @@ public sealed class AzureTableStorageJournalSettings
         public static AzureTableStorageJournalSettings Create(Config config)
         {
             var connectionString = config.GetString("connection-string");
-            var tableName = config.GetString("table-name");
+            var tableName = config.GetString("table-name", DefaultTableName);
```

Comment:
I realize that easier way would be to set this default names in default hocon configuration - but this way users will be able to get name value from `DefaultTableName` constant in runtime. Which might be useful.

Reply:
These values should be stored in HOCON.